grommet-toolbox.config.js

```diff
@@ -26,6 +26,15 @@ export default {
         path.resolve(__dirname, 'src/scss'),
         path.resolve(__dirname, 'node_modules')
       ]
+    },
+    module: {
+      loaders: [
+        {
+          test: /\.jsx?$/,
+          loader: 'babel-loader',
+          include: /node_modules\/react-gravatar/
```

Comment:
i think that would be the best solution, rather than adding another webpack loader for 1 specific module.